src/addrman.cpp

```diff
@@ -733,3 +736,22 @@ std::vector<bool> CAddrMan::DecodeAsmap(fs::path path)
     }
     return bits;
 }
+
+void LoadAddrman(std::unique_ptr<CAddrMan>& addrman, const ArgsManager& args)
+{
+    assert(!addrman);
```

Comment:
Maybe return `std::unique_ptr<CAddrMan>`? (or an optional, if it can fail, but doesn't seems so)

Reply:
Done